src/index.js

```diff
@@ -1,29 +1,22 @@
 import './style.css';
 import { saveLocalStorage, updateCompleted } from './active';
+import {
+  editTodo, removeTodos, addTodo, clearCompletedTodos,
+} from './add';
 
-let items = [
-  {
-    description: 'wash the dishes',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'complete To Do List project',
-    completed: false,
-    index: 1,
-  },
-];
+let itemsArray = [];
 
 const storageData = JSON.parse(localStorage.getItem('toDoStorage'));
 if (storageData) {
-  items = storageData;
+  itemsArray = storageData;
 }
 
 const todos = document.querySelector('.todos');
-items.forEach((item) => {
+itemsArray.forEach((item) => {
   const html = `<input type="checkbox">
-  <input value=${item.description} type="text"></input>
-  <i class="fas fa-ellipsis-v"></i>`;
+  <input class="edit" value=${item.description} type="text"></input>
+  <i class="fas fa-ellipsis-v vertical"></i>
+  <i class="fas fa-trash-alt trash"></i>`;
```

Comment:
- [ ] We can not delete a task because there is no trash icon on the UI for each task. Let's fix that

![Screenshot from 2021-12-08 17-35-23](https://user-images.githubusercontent.com/40503704/145247877-aa689849-3f21-4060-bc47-7d8a6a777409.png)